docs/release/3.4/release-3-4-8.md

```diff
@@ -0,0 +1,83 @@
+OSG Software Release 3.4.8
+==========================
+
+**Release Date**: 2018-02-07
+
+Summary of changes
+------------------
+
+This release contains:
+
+-   Critical GlideinWMS bug fix for heavily loaded frontends (<https://cdcvs.fnal.gov/redmine/issues/18748>)
+
+These [JIRA tickets](https://jira.opensciencegrid.org/issues/?jql=project%20%3D%20SOFTWARE%20AND%20fixVersion%20%3D%203.4.8%20ORDER%20BY%20priority%20DESC%2C%20key%20DESC) were addressed in this release.
+
+!!! note "Notes"
+    -   OSG 3.4 contains only 64-bit components.
+    -   StashCache is supported on EL7 only.
+    -   xrootd-lcmaps will remain at 1.2.1-2 on EL6.
+    -   OSG PKI tools now create service certificate files where the service tag is separated from the hostname with an underscore (`_`). This new format first appeared in OSG PKI tools version 2.1.2.
+
+Detailed changes are below. All of the documentation can be found [here](/index.md).
+
+Known Issues
+------------
+
+-   Because the Gratia system has been turned off, RSV emits the following warning: `WARNING: Gratia server gratia-osg-prod.opensciencegrid.org:80 not responding to obtain last contact details`. This warning can safely be ignored.
+
+Updating to the new release
+---------------------------
+
+To update to the OSG 3.4 series, please consult the page on [updating between release series](/release/release_series#updating-from-osg-31-32-33-to-33-or-34).
+
+### Update Repositories
+
+To update to this series, you need [install the current OSG repositories](/common/yum#install-osg-repositories).
+
+### Update Software
+
+Once the new repositories are installed, you can update to this new release with:
+
+``` console
+root@host # yum update
+```
+
+!!! note "Notes"
+    -   Please be aware that running `yum update` may also update other RPMs. You can exclude packages from being updated using the `--exclude=[package-name or glob]` option for the `yum` command.
+    -   Watch the yum update carefully for any messages about a `.rpmnew` file being created. That means that a configuration file had been edited, and a new default version was to be installed. In that case, RPM does not overwrite the edited configuration file but instead installs the new version with a `.rpmnew` extension. You will need to merge any edits that have made into the `.rpmnew` file and then move the merged version into place (that is, without the `.rpmnew` extension). Watch especially for `/etc/lcmaps.db`, which every site is expected to edit.
```

Comment:
Should we still give the warning about `/etc/lcmaps.db.rpmnew` now that people should be using osg-configure to configure it?

Reply:
Definitely.  I strongly doubt the majority of folks utilize `osg-configure` on things like the GridFTP or Xrootd hosts.